JapaneseFilter/sections/specific.txt

```diff
@@ -42,6 +42,8 @@ aviationwire.jp,getnavi.jp,entamerush.jp,gourmetpress.net,career-find.jp,mainich
 bridge-salon.jp##div[id^="jp_ir_"][style]
 blkt.net##.ad001
 blkt.net##.mkjAd
+locari.jp##.footer__banner
```

Comment:
This is PR for advertiser who may want their ads to be put on the site. Should we block this on main list? @Alex-302 @zloyden

Reply:
I don't think that we should remove it from the footer.